OpenSim/Simulation/Model/Model.cpp

```diff
@@ -2071,7 +2096,7 @@ void Model::realizeReport(const SimTK::State& state) const
 void Model::computeStateVariableDerivatives(const SimTK::State &s) const
 {
     try {
-        realizeAcceleration(s);
+        extendRealizeAcceleration(s);
```

Comment:
Do you know how this change happened? Did you find-replace? Does Atom have an AI?